integration-tests/tests/multichain/mod.rs

```diff
@@ -49,3 +56,65 @@ async fn test_triples_and_presignatures() -> anyhow::Result<()> {
     })
     .await
 }
+
+#[test(tokio::test)]
+async fn test_signature() -> anyhow::Result<()> {
+    with_multichain_nodes(3, |ctx| {
+        Box::pin(async move {
+            // Wait for network to complete key generation
+            let state_0 = wait_for::running_mpc(&ctx, 0).await?;
+            assert_eq!(state_0.participants.len(), 3);
+
+            let worker = &ctx.nodes.ctx().worker;
+            let (account_id, secret_key) = worker.dev_generate().await;
+            worker
+                .create_tla(account_id.clone(), secret_key.clone())
+                .await?
+                .into_result()?;
+            let payload: [u8; 32] = rand::thread_rng().gen();
+            let outcome = ctx
+                .rpc_client
+                .send_tx(
```

Comment:
Since the protocol feature is yet to be implemented, what is our plan here (for this PR and first testnet release)?

Reply:
We are just making a separate unrelated transaction with the response